roads.mss

```diff
@@ -2888,6 +2895,7 @@ tertiary is rendered from z10 and is not included in osm_planet_roads. */
       text-face-name: @book-fonts;
       text-halo-radius: @standard-halo-radius;
       text-halo-fill: @tertiary-fill;
+      text-repeat-distance: @text-repeat-distance;
```

Comment:
This code is not working properly in my tests - see my previous comment on `text-repeat-distance` values.